pages/login.tsx

```diff
@@ -78,13 +79,14 @@ export const Login: React.FC<LoginFormProps> = ({
 }
 
 const LoginPage: React.FC = () => {
+  const router = useRouter()
   const [loginErrors, setLoginErrors] = useState<string[]>([])
   const [loginUser, { data, error }] = useMutation(LOGIN_USER)
```

Comment:
Found a solution for the stale cache after login. See the Apollo links below for a more detailed explanation.

After completing the login mutation, we can tell Apollo to refetch a query and update its cache using the `refetchQueries` option. So instead of needing to check to server to update the cache with that fetch policy, we can simply update it after login since a session will now exist. And also no more loading spinner when on the curriculum page 🙂

Reference: 
- [When To Use Refetch Queries in Apollo Client](https://www.apollographql.com/blog/when-to-use-refetch-queries-in-apollo-client/)
- [Advanced topics on caching - Rerunning queries after a mutation](https://www.apollographql.com/docs/react/caching/advanced-topics/#rerunning-queries-after-a-mutation)

```suggestion
  const [loginUser, { data, error }] = useMutation(LOGIN_USER, {
    refetchQueries: [
      {
        query: GET_APP
      }
    ]
  })
```